.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
     - $HOME/.sbt
     - $HOME/.ivy2
 script:
-  - sbt "set parallelExecution in Test := false" clean coverage doc assembly
+  - sbt clean scapegoat doc coverage package
```

Comment:
But now test fails due to low coverage https://travis-ci.org/janisz/marathon/builds/228659037

```
[info] Statement coverage.: 21.22%
[info] Branch coverage...: 19.14%
[info] Coverage reports completed
[error] Coverage is below minimum [21.22% < 75.0%]
```